examples/moodle-presslabs-stash/steps.txt

```diff
@@ -0,0 +1,172 @@
+     - kubectl create -f moodle1.yaml
+
+   - Wait till Moodle Pod is ready (It will take about 5/6 minutes for Moodle Pod to become ready)
+     - kubectl get pods -n namespace1
+
+   - Login to Moodle Instance
+       - Update /etc/hosts with <minikube ip or cluster node ip> moodle1. Example:
+         - 192.168.99.100 moodle1
+         - You can find minikube ip using: "minikube ip" command
+         - If using Managed Kubernetes, find out the cluster node ip by refering to appropriate documentation
+           from the provider.
+         - Retrieve Moodle instance's admin password
+	   - kubectl describe moodles moodle1 -n namespace1
+             - Note down the secret name and Moodle instance URL
+         - kubectl get secret moodle1 -n namespace1 -o jsonpath="{.data.adminPassword}" | base64 --decode
+         - Navigate to the URL of moodle1 instance (available in the output of 'describe' command)
+           - Login using 'admin' as username and password retrieved earlier from 'kubectl get secret' command
+    - Check installed plugins
+      - As part of creating moodle instance, we install the 'profilecohort' plugin.
+        Check the custom resource specification moodle1.yaml to see this definition.
+        - Once logged in to the Moodle instance, hit 'Site Administration'.
+          This will show a message to update Moodle database for 'profilecohort' plugin.
+        - Select that option to complete Plugin installation.
+          You should see a 'success' message when the plugin is installed.
+        - Hit the 'contine' button. This will take you to the main screen.
+          Fill out the profile and then hit 'update profile'.
+        - Then
+          - Navigate to -> Administration -> Plugins -> Plugins Overview
+          - You should see 'profilecohort' plugin in the 'Additional plugins' list
+    - Create a new course
+        - Go to Site home
+        - Add a new course
+        - Fill in the fields and
+            Upload an image > Upload a file
+        - Save and return
+
+2) See the composition of Moodle instance, checking for deployment name, should be moodle1
+    - kubectl get --raw "/apis/kubeplus.cloudark.io/v1/composition?kind=Moodle&instance=moodle1&namespace=namespace1" | python -mjson.tool
+
+3) Add a label to the Moodle deployment
+    - kubectl label -n namespace1 deployments moodle1 app=moodle-deploy
+
+4) Add base64 encoded access keys to s3-secret.yaml
+    Edit your keys s3-secret.yaml
+
+    for AWS_ACCESS_KEY_ID:
+    - echo your_access_key | base64
+    for AWS_SECRET_ACCESS_KEY:
+    - echo your_secret_access_code | base64
+    for RESTIC_PASSWORD:
+    - echo 'canbeanything' | base64
+
+5) Apply secret
+    - kubectl apply -f s3-secret.yaml
+
+6) Start taking backups
+    - kubectl apply -f restic-moodle.yaml
+
+7. Validate that it is taking backups (may take one minute)
+    - aws s3 ls s3://stash-testing34/stash1/deployment/moodle1/
+
+
+Recover a snapshot
+-----------
+
```

Comment:
We will need to first cd into moodle-recovery folder, right?

cd moodle-recovery

Reply:
I am sending a new PR for this

Reply:
Sounds good